index.html

```diff
@@ -148,87 +175,118 @@ <h2 class="head2">Tonic</h2>
+                <h2 class="head2 statusd">Uber Navigation</h2>
+                <div class="up unique3 unique6">
+                    <div class="p1 status">CANOPY</div>
+                    <div class="p1 statusd">Uber</div>
                     <div class="counter"></div>
-                    <div class="p2">Back End Dev</div>
+                    <div class="p2 status">Back End Dev</div>
+                    <div class="p2 statusd lead-dev">Lead Developer</div>
                     <div class="counter"></div>
-                    <div class="p3">2015</div>
+                    <div class="p3 status">2015</div>
+                    <div class="p3 statusd">2018</div>
+                </div>
                 </div>
             </section>
-            <img src="" alt="">
             <section>
-                <p class="p4">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                <ul class="middle">
+                <p class="p4 status">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <p class="p4 p4-unique2 statusd">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
+                <div class="skill-project">
+                <ul class="middle middled">
                     <li class="back-link1 back-link">
                         <div class="link1">html</div>
                     </li>
+                    <li class="back-link4 back-link statusd">
+                        <div class="link4 statusd">Ruby on rails</div>
+                    </li>
                     <li class="back-link2 back-link">
                         <div class="link2">css</div>
                     </li>
                     <li class="back-link3 back-link">
                         <div class="link3">javaScript</div>
                     </li>
                 </ul>
-                <button class="see-project1">
+                <button class="see-project1 unique4">
                     <div class="see-project">See Project</div>
                 </button>
+                </div>
             </section>
         </div>
         <section>
             <div></div>
             <div></div>
         </section>
     </section>
-    </div>
-    <section class="section6">
+    <section class="section2 section2d">
+        <div class="aboutd">
         <h2 class="h">About <br>Myself</h2>
-        <p class="info11">Hello I’m a software developer! I can help you build a product , feature or website Look
+        <p class="info11 info11d">Hello I’m a software developer! I can help you build a product , feature or website Look
             through some of my work and experience! If you like what you see and have a project you need coded, don’t
             hestiate to contact me.</p>
         <p class="lc info2">LET’S CONNECT</p>
-        <footer class="foot">
+        <div class="socials">
```

Comment:
- [ ] Kindly adjust the width of this section to match the design in Figma, it should be wider and it should have more padding on the sides

## Your design:
![image](https://user-images.githubusercontent.com/50721479/206857503-e5083e40-58d5-49b6-b3e3-a87a4d1f04c5.png)

## Figma desgn:
![Uploading image.png…]()